packaging/grafana-agent-flow/windows/install_script.nsis

```diff
@@ -51,6 +52,7 @@ Section "install"
   ${GetParameters} $PassedInParameters
   ${GetOptions} $PassedInParameters "/DISABLEPROFILING=" $DisableProfiling
   ${GetOptions} $PassedInParameters "/DISABLEREPORTING=" $DisableReporting
+  ${GetOptions} $PassedInParameters "/ENVIRONMENT=" $Environment
```

Comment:
Is it possible to pass more than one environment variable with this? It seems like this is restricted to a single variable, which is a bit weird to me.

Reply:
Yes, see docs above

`/ENVIRONMENT="KEY=VALUE\0KEY2=VALUE2"`

The REG_MULTI_SZ typ is a string with null byte separator. The value is passed one by one to [`reg add`](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/reg-add#examples) command. 


I agree, this feels a bit unintuitive on first view, but from technical point of view it has some benefits. If we decide to use `;` as separator, then an additional logic is required to escape separator. And the logic needs to be written in NSIS script.